app/konnect/runtime-manager/composite-runtime-groups/invariances.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Invariances in runtime groups
```

Comment:
todo: are we sticking with invariance or calling it something else?
- conflicts?
- deviations?
- notable differences?

Reply:
We are aligned to use the term "conflict"